Formula/simgrid.rb

```diff
@@ -15,6 +15,8 @@ class Simgrid < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "python3" => :build
```

Comment:
are you sure this is a build-time only requirement? That is very rare.

Reply:
Sorry. My mistake. Copied the whole line.